MaintenanceDashboard.Data/Domain/ReceivedPaddle.cs

```diff
@@ -5,21 +5,27 @@ namespace MaintenanceDashboard.Data.Domain
 {
     public class ReceivedPaddle
     {
-        [Key]
         public int Id { get; set; }
+        
         [Required]
         public string ReceivingEmployee { get; set; }
-        [ForeignKey("Paddle")]
```

Comment:
tak jak wyżej, nie sugeruj się tym, nie wiem czy EF z .NET Frameworka ogarnia to tak jak ten z .NET Core